extensions/sourcegraph/package.json

```diff
@@ -120,11 +120,12 @@
   ],
   "dependencies": {
     "@bobheadxi/node-eventsource-http2": "^1.1.1",
-    "@raycast/api": "^1.27.0",
+    "@raycast/api": "^1.27.1",
     "luxon": "^2.3.0",
-    "node-fetch": "^3.0.0",
+    "node-fetch": "^3.2.0",
+    "react": "^17.0.2",
```

Comment:
You can remove react and react-reconciler as dependencies since they will be provided at runtime. It should be enough to have "@types/react" in the devDependencies

Reply:
Gotcha, thanks! fixed :)